### PR TITLE
master function unable to detect connection to master on ipv6 address

### PR DESCRIPTION
**Pull request by trootnev**
### What does this PR do?
If minion and master are connected only using ipv6 addresses then this function always returns false marking minion disconnected. This causes minion stopping returning any results back to master after 1 hour because by default once in an hour the scheduled job calls state.master.
The root cause is that the function is using netstat command output with parameter -n -p TCP. It shows only IPv4 connections. To see both IPv4 and IPv6 we need to omit -p parameter completely and filter out non-TCP connections afterwards. Also we need to remove square brackets in IPv6 addresses returned as we will compare those to the ones without brackets later in code.

### What issues does this PR fix or reference?
Fixes:

### Previous Behavior
Every time the master function run on windows minion with only IPv6 address connected to salt-master it (function) used to conclude that minion is disconnected and responses were not sent since that moment until minion restart.

### New Behavior
Minion connection status gets evaluated correctly on both IPv6 and IPv4 addresses

### Merge requirements satisfied?
No. Sorry, Windows environment is used, test setup is too complicated for Windows. This change is very easy to check and I checked it in my test environment manually.
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Comment by trootnev** [comment]
something's clearly wrong with those tests... Linux-related tests are failing on something unrelated to what has been changed and the module itself is aiming windows systems only.

**Review by cmcmarrow — CHANGES_REQUESTED** [review]
@trootnev thanks for the PR! If you need help with pre-commit please read https://pre-commit.com/. Also this PR still needs a test.

File: salt/modules/win_status.py [hunk 1]
```diff
@@ -539,21 +539,21 @@ def _win_remotes_on(port):
...
         for line in lines:
-            if "ESTABLISHED" not in line:
+            if "ESTABLISHED" not in line or "TCP" not in line:
                 continue
             chunks = line.split()
             remote_host, remote_port = chunks[2].rsplit(":", 1)
             if int(remote_port) != port:
                 continue
-            remotes.add(remote_host)
+            remotes.add(remote_host.replace('[', '').replace(']', ''))
```
Comment (waynew) [inline salt/modules/win_status.py:556]: **question** would it be a good idea to use the [`ipaddress` module](https://docs.python.org/3/library/ipaddress.html) here, or is that overkill?

**Comment by garethgreenaway** [comment]
@trootnev Following up on this one and checking that you saw the comments above, as well as the request for a automated test.  Thanks!

**Comment by dwoz** [comment]
Closing this due to inactivity. Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

**Pull request closed without merging**